app/assets/stylesheets/components/_alert.scss

```diff
@@ -23,3 +23,7 @@
     transform: translateY(-50%);
   }
 }
+
+.usa-alert__text > p:last-child {
+  margin-bottom: 0;
+}
```

Comment:
Typically, the paragraph should be assigned with the class `.usa-alert__text`, which [resets vertical margins](https://github.com/uswds/uswds/blob/30c61f34288bc706961b12b536fdc57537026385/packages/uswds-core/src/styles/mixins/helpers/alert-status-styles.scss#L47). Since we have a tendency to show multiple paragraphs of text in an alert, the way we support this is to render the `.usa-alert__text` element as a `div` wrapper. We want to allow for the default paragraph margin _between_ paragraphs, but remove it from the last paragraph in the container.